proto/internalapi/central/compliance_operator.proto

```diff
@@ -235,6 +235,7 @@ message ComplianceOperatorScanStatusV2 {
   int64 current_index = 4;
   string warnings = 5;
   int64 remaining_retries = 6;
+  google.protobuf.Timestamp last_start_time = 9;
```

Comment:
When the proto engine was updated it was decided (I think) that we should keep fields in numerical order to make it easier to determine what is next.  So recommend you do that here and elsewhere.

Reply:
Done